src/librustc/middle/ty.rs

```diff
@@ -815,6 +820,11 @@ bitflags! {
         const HAS_TY_ERR         = 1 << 6,
         const HAS_PROJECTION     = 1 << 7,
         const HAS_TY_CLOSURE     = 1 << 8,
+
+        // true if there are "names" of types and regions and so forth
+        // that are local to a particular fn
+        const HAS_LOCAL_NAMES   = 1 << 8,
```

Comment:
From other parts of the patch, it looked like a rename from `HAS_TY_CLOSURE` to `HAS_LOCAL_NAMES`.